### PR TITLE
Fix log messages which are not format strings

### PR DESCRIPTION
**Pull request by ksobolew**
## Description

See also: 2ec05295229ba1851f7346a3654c0342423ef658

This is currently not enforced in Error Prone because it requires annotating more methods in `airlift:log` with `@FormatMethod`, but that's hard to do, because it will change their semantics.

## Additional context and related issues

Extracted from #14933.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java [hunk 1]
```diff
@@ -571,7 +571,7 @@ private <T> T executeWithSession(SessionCallable<T> sessionCallable)
                     throw e;
                 }
                 long delay = Math.min(schedule.nextDelay().toMillis(), timeLeft);
-                log.warn(e.getMessage());
```
Comment (findepi) [inline plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java:574]: per https://github.com/trinodb/trino/pull/14933#discussion_r1095750818 i don't think we should do a change here

**Review by findepi — APPROVED** [review]
%

File: plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java (hunk 1 above)
Comment (ksobolew) [reply plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java]: Reverted

**Review by findepi — APPROVED** [review]
thanks!

**Review by raunaqmorarka — APPROVED** [review]

**Pull request merged**